randomizer.py

```diff
@@ -825,36 +851,48 @@ def potionCards(self):
 Allies.AddCards(
     [
         "Bauble",
-        "Sycophant",
-        "Townsfolk: Town Crier + Blacksmith + Miller + Elder",
-        "Augers: Herb Gatherer + Acolyte + Sorceress + Sibyl",
-        "Clashes: Battle Plan + Archer + Warlord + Territory",
-        "Forts: Tent + Garrison + Hill Fort + Stronghold",
-        "Merchant Camp",
-        "Importer",
-        "Odysseys: Old Map, Voyage, Sunken Treasure, Distant Shore",
-        "Sentinel",
-        "Underling",
-        "Wizards: Student, Conjurer, Sorcerer, Lich",
-        "Broker",
-        "Carpenter",
-        "Courier",
-        "Innkeeper",
-        "Royal Galley",
-        "Town",
-        "Barbarian",
-        "Capital City",
         "Contract",
-        "Emissary",
-        "Galleria",
-        "Guildmaster",
-        "Highwayman",
-        "Hunter",
-        "Modify",
-        "Skirmisher",
-        "Specialist",
-        "Swap",
-        "Marquis",
+        {"name": "Sycophant", "types": {Action}},
+        {
+            "name": "Townsfolk: Town Crier + Blacksmith + Miller + Elder",
+            "types": {Action},
+        },
+        {
+            "name": "Augers: Herb Gatherer + Acolyte + Sorceress + Sibyl",
+            "types": {Action},
+        },
+        {
+            "name": "Clashes: Battle Plan + Archer + Warlord + Territory",
```

Comment:
Is Territory eligible for Obelisk?

Reply:
Yes, the eligibility is based on the randomizer card, which is an action for all of the 4x split piles.